opencti-platform/opencti-graphql/tests/01-unit/database/rss-test.js

```diff
@@ -0,0 +1,324 @@
+import { describe, expect, it } from 'vitest';
+import TurndownService from 'turndown';
+import { rssDataParser } from '../../../src/manager/ingestionManager';
+
+// region https://www.oracle.com/ocom/groups/public/@otn/documents/webcontent/rss-otn-sec.xml
```

Comment:
Maybe the initialization of those variables could be extracted to a single file or file by variable.

Reply:
Extracted to xml files in data